src/Sylius/Behat/Client/ApiPlatformClient.php

```diff
@@ -94,6 +102,21 @@ public function delete(string $resource, string $id): void
         $this->request('DELETE', sprintf('/new-api/%s/%s', $resource, $id), []);
     }
 
+    public function filter(string $resource): void
+    {
+        $path = sprintf('/new-api/%s?', $resource);
+        foreach ($this->filters as $key => $value) {
+            if (substr($path, -1) === '?') {
+                $path = sprintf('%s%s=%s', $path, $key, $value);
+                continue;
+            }
+
+            $path = sprintf('%s&%s=%s', $path, $key, $value);
```

Comment:
Maybe this would work better https://www.php.net/manual/en/function.http-build-query.php?